Description: This makes sure that the CXXFLAGS given by the
 Debian build system are not overwritten, but prepended. Additionally
 unwanted CXXFLAGS are removed.
Author: Ruben Undheim <ruben.undheim@gmail.com>

Index: icestorm/icemulti/Makefile
===================================================================
--- icestorm.orig/icemulti/Makefile	2015-10-08 08:36:26.843035669 +0200
+++ icestorm/icemulti/Makefile	2015-10-08 08:36:58.549608012 +0200
@@ -1,7 +1,7 @@
 # CXX = clang
 CXX ?= clang
 LDLIBS = -lm -lstdc++
-CXXFLAGS = -MD -O0 -ggdb -Wall -std=c++11
+CXXFLAGS += -Wall -std=c++11
 CC = $(CXX)
 DESTDIR = /usr/local
 
Index: icestorm/icepack/Makefile
===================================================================
--- icestorm.orig/icepack/Makefile	2015-10-08 08:36:26.843035669 +0200
+++ icestorm/icepack/Makefile	2015-10-08 08:36:46.837429230 +0200
@@ -1,7 +1,7 @@
 # CXX = clang
 CXX ?= clang
 LDLIBS = -lm -lstdc++
-CXXFLAGS = -MD -O0 -ggdb -Wall -std=c++11 -I/usr/local/include
+CXXFLAGS += -Wall -std=c++11
 CC = $(CXX)
 DESTDIR = /usr/local
 
Index: icestorm/iceprog/Makefile
===================================================================
--- icestorm.orig/iceprog/Makefile	2015-10-08 08:36:26.843035669 +0200
+++ icestorm/iceprog/Makefile	2015-10-08 08:37:21.418286898 +0200
@@ -1,6 +1,6 @@
 # CC = clang
 LDLIBS = -L/usr/local/lib -lftdi -lm
-CFLAGS = -MD -O0 -ggdb -Wall -std=c99 -I/usr/local/include
+CFLAGS += -Wall -std=c99
 DESTDIR = /usr/local
 
 all: iceprog
